app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -371,25 +465,59 @@ export default {
         });
       }
     },
-    updateVersions() {
+    canViewSubmissions() {
```

Comment:
This should be a computed function instead of a method function. Also, this shouldn't be necessary as you need the role of submission viewer to access this page.